cluster/ci/config/prow/openshift/jenkins-config-updater/config-ci.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jenkins-config-updater
+data:
+  update.yaml: |-
+    targets:
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/deployment.yaml
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/build.yaml
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/config-ci.yaml
+    - projects/origin-release/pipeline.yaml
+    - projects/prometheus/prometheus.yaml
+    - projects/prometheus/node-exporter.yaml
+    - projects/kube-state-metrics/pipeline.yaml
+    - projects/gcsweb/pipeline.yaml
+    - projects/oauth-proxy/pipeline.yaml
+    - cluster/ci/config/submit-queue/submit_queue.yaml
+    - cluster/ci/config/submit-queue/submit_queue_openshift_ansible.yaml
+    - cluster/ci/config/submit-queue/submit_queue_origin_aggregated_logging.yaml
+    - cluster/ci/config/submit-queue/submit_queue_origin_web_console.yaml
+    - cluster/ci/config/prow/prow_crd.yaml
```

Comment:
This means that the plugin needs to have cluster admin access.

Reply:
Same for `prowjob_access.yaml` below.

Reply:
Hm, it is OK I guess if we can't update these two automatically, then.

Reply:
They won't be changing often